### ISSUE TITLE
I an’t find this novel, can you please add it

### ISSUE THREAD

**Issue by lupinsanigata** [MakeYourLifeEasier/novel-ebook.com#149]
This novel looks great can you please add it : https://www.webnovel.com/book/walker-of-the-worlds_17307252205313105
Thanks in advance.

**Comment by Emasoft** [comment]
Since there is already this thread, I’ll add this novel to the list of missing ones:
https://www.lightnovelpub.com/novel/regressor-instruction-manual-17031322

**Comment by lupinsanigata** [comment]
thanks

**Comment by LKADity** [comment]
I also request for this novel to be added
https://www.novelupdates.com/series/little-tyrant-doesnt-want-to-meet-with-a-bad-end/

**Comment by hobby90** [comment]
I'll also request for this novel, https://www.novelupdates.com/series/this-overlord-doesnt-care-about-anything/

Also, this one, https://www.google.com/amp/s/m.webnovel.com/amp/book/18549008105094005

And lastly, https://www.novelupdates.com/series/i-am-a-gao-fushuai-villain/

**Comment by hobby90** [comment]
Just wondering if these novels are going to be added? Sorry if it's an inconvenience to do so!

**Comment by anszzxzz** [comment]
many available novel gone missing,is there some reason

**Comment by aki277** [comment]
yeah, I have many novels which I downloaded previously, but I am unable to find them now.

please fix this problem.

**Comment by lwneater** [comment]
Hello, can I request for this novel https://www.lightnovelpub.com/novel/nanomancer-reborn-ive-become-a-snow-girl-04092103

**Comment by Emasoft** [comment]
Another missing novel:

The Way Of The Dao

https://www.scribblehub.com/series/186389/the-way-of-the-dao/

or 

https://www.webnovel.com/book/the-way-of-the-dao_16874262805580005

**Comment by JadeEmpress** [comment]
Hello, thanks for all the work. This "Silent Hacker"  is an amazing novel. Can you please add it: 

https://m.webnovel.com/book/silent-hacker_13515430106134305

**Comment by daff777** [comment]
Hi! How are you? Hope you are doing great :)
I would like to request to have this novel added if possible "I BECOME A BURDENSOME CHILD AFTER TRANSMIGRATING". Thank you!
https://novelfull.com/i-become-a-burdensome-child-after-transmigrating.html

**Comment by NastjaK666** [comment]
Hello, can you please add Kill The Hero from https://woopread.com/series/kill-the-hero/

I tried to make a file myself, but it seems like there is a limit for scrapers or some kind of simple block( 

Thank you!
